aviary/subsystems/propulsion/prop_performance.py

```diff
@@ -0,0 +1,231 @@
+            units="unitless",
+            desc="square of DiamNac/DiamProp",
+        )
+
+        self.blockage_factor_interp.add_input(
+            "equiv_adv_ratio",
+            0.0,
+            training_data=[0., 0.5, 1.0, 2.0, 3.0, 4.0, 5.0],
+            units="unitless",
+            desc="square of DiamNac vs DiamProp",
+        )
+
+        self.blockage_factor_interp.add_output(
+            "blockage_factor",
+            0.765,
+            units="unitless",
+            desc="blockage factor",
+            training_data=np.array(
+                [[1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0],
+                 [0.992, 0.991, 0.988, 0.983, 0.976, 0.970, 0.963],
+                 [0.986, 0.982, 0.977, 0.965, 0.953, 0.940, 0.927],
+                 [0.979, 0.974, 0.967, 0.948, 0.929, 0.908, 0.887],
+                 [0.972, 0.965, 0.955, 0.932, 0.905, 0.872, 0.835],
+                 [0.964, 0.954, 0.943, 0.912, 0.876, 0.834, 0.786],
+                 [0.955, 0.943, 0.928, 0.892, 0.848, 0.801, 0.751],
+                 [0.948, 0.935, 0.917, 0.872, 0.820, 0.763, 0.706],
+                 [0.940, 0.924, 0.902, 0.848, 0.790, 0.726, 0.662]]
+            ),
+        )
+
+        self.add_subsystem(
+            name='FT',
+            subsys=om.ExecComp(
+                'install_loss_factor = 1 - blockage_factor',
+                blockage_factor={'units': 'unitless', 'val': np.zeros(nn)},
+                install_loss_factor={'units': 'unitless', 'val': np.zeros(nn)},
+            ),
+            promotes_inputs=["blockage_factor"],
+            promotes_outputs=["install_loss_factor"],
+        )
+
+
+class PropPerf(om.Group):
+    """
+    Computation of propeller thrust coefficient based on Hamilton Standard.
+    The installation loss factor is either a user input or computed internally.
+    """
+
+    def initialize(self):
+        self.options.declare(
+            'num_nodes', types=int,
+            desc='Number of nodes to be evaluated in the RHS')
+        self.options.declare(Aircraft.Engine.NUM_BLADES, default=2,
+                             desc='number of blades per propeller')
+        self.options.declare(
+            Aircraft.Design.COMPUTE_INSTALLATION_LOSS, types=bool,
+            desc='Flag to compute installation factor')
+        self.options.declare(
+            'aviary_options', types=AviaryValues,
+            desc='collection of Aircraft/Mission specific options')
+
+    def setup(self):
+        options = self.options
+        nn = options['num_nodes']
+        aviary_options = options['aviary_options']
+        compute_installation_loss = aviary_options.get_val(
+            Aircraft.Design.COMPUTE_INSTALLATION_LOSS)
+        num_blades = aviary_options.get_val(Aircraft.Engine.NUM_BLADES)
+
+        # JK NOTE it looks like tloss can be its own component. Optionally loaded? Or install_loss_factor is an override value? Might need to talk with Ken
```

Comment:
has this been addressed by the the InstallLoss component?

Reply:
yes, removed note